routes/web.php

```diff
@@ -16,6 +16,9 @@
 // Marketing page routes
 Route::view('/', 'marketing.home');
 
+// Privacy page routes
+Route::view('/privacy', 'privacy.privacypage');
+
 Route::name('auth.')
     ->group(function () {
         Route::get('/login', [LoginController::class, 'create'])
```

Comment:
For some reason I can't comment at the end of the file..
In the Route::name('admin') group right at the bottom it looks like this is missing, maybe you forgot to commit it? - I have been getting a 404 when trying to unpair. 
          Route::post('/delete', [AdminDashboardController::class, 'destroy'])
                ->name('dashboard.destroy');